### DISCUSSION TITLE
InputNumber spinner buttons layout change

### DISCUSSION THREAD

**Discussion by jnlinn** [primefaces/community#2416] [PrimeReact] [answered]
Hello, I just updated to PrimeReact 10.8.0 and noticed all of the inputNumbers with spinner buttons in my application layouts are changed. Here is an image from the PrimeReact demo page:

![2024-07-29 14_46_14-React InputNumber Component — Mozilla Firefox](https://github.com/user-attachments/assets/39808b8b-c2f7-4cd0-bb7b-3e27857e47ab)

When was this changed or is this a glitch? The "horizontal" button layout should display the buttons on each end of the input field.

**Comment by nitrogenous** [comment]
Here's a more polished and formal response for you:

Thank you for bringing this to our attention. We will address this issue as soon as possible and will release a bug fix within this week.

**Reply by nitrogenous** [reply, marked as answer]
https://github.com/primefaces/primereact/issues/6953
